### ISSUE TITLE
[Bug]: POS app icons are all black when light theme is active

### ISSUE THREAD

**Issue by bolatovumar** [btcpayserver/btcpayserver#4708]
### What is your BTCPay version?

v1.7.12

### How did you deploy BTCPay Server?

LunaNode

### What happened?

See this screenshot:

![Capture](https://user-images.githubusercontent.com/1934678/221475209-53817ec8-a72e-405e-9f10-794fe068dccb.PNG)

POS app type icons are just all black with light theme active. They look as expected when dark theme is active. I couldn't figure out why this is happening. I tried clearing my cache just to make sure I didn't have some old cached SVG.

### How did you encounter this bug?

Just went to the POS app on my store.

### What browser do you use?

Chrome 110

### Are you sure this is a bug report?

- [X] I confirm this is a bug report

**Comment by pavlenex** [comment]
Can you replicate it on https://mainnet.demo.btcpayserver.org/

**Comment by dennisreimann** [comment]
Me neither, works for me across browsers. @bolatovumar does it work on this overview page for you? https://design.btcpayserver.org/design/icons/

**Comment by bolatovumar** [comment]
> Me neither, works for me across browsers. @bolatovumar does it work on this overview page for you? https://design.btcpayserver.org/design/icons/

@dennisreimann nope, looks fine there

**Comment by bolatovumar** [comment]
> Can you replicate it on https://mainnet.demo.btcpayserver.org/?
> 
> I couldn't.

Neither can I but it happens consistently in my store. Also, I tried logging in incognito mode and the issue still reproed.

**Comment by bolatovumar** [comment]
I think I figured out why my icons show up as all black. My `icon-sprite.svg` file is different from what's in the repo and it's actually the older version before this PR was merged: https://github.com/btcpayserver/btcpayserver/pull/4632

Because my SVG is missing `fill="none"` for all icons they show up as black. Adding `fill="none"` through CSS fixes the issue.

The above PR is not part of `v1.7.12`, however, so others who have the same version (commit hash `9344113ae`) would have the same issue probably?
